tdrs-backend/tdpservice/users/test/test_auth_check.py

```diff
@@ -71,3 +71,12 @@ def test_auth_check_returns_user_stt(api_client, user):
     serializer = UserProfileSerializer(user)
     response = api_client.get(reverse("authorization-check"))
     assert response.data["user"]["stt"] == serializer.data["stt"]
+
+
+@pytest.mark.django_db
+def test_auth_check_inactive_account(api_client, inactive_account_user):
+    """If user is deactivated, return a response indicating the user is inactive."""
+    api_client.login(username=inactive_account_user.username, password="test_password")
+    response = api_client.get(reverse("authorization-check"))
+    assert response.data["authenticated"] is False
+    assert response.data["inactive"] is True
```

Comment:
Assert that an inactive user returns a response indicating as such, so the UI can use this to render the alert.